src/components/Header/HeaderInfoDialog.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
 import RegistryInfoDialog from '../RegistryInfoDialog';
-import RegistryInfoContent from '../RegistryInfoContent';
+import { RegistryInfoContent } from '../RegistryInfoContent';
```

Comment:
Hmm... I would rather leave it as a default import as it it the main code of `RegistryInfoContent`

Reply:
generally speaking, using default imports/exports can lead to issues. Is there a reason why we prefer to use them vs named imports/exports?

https://basarat.gitbooks.io/typescript/docs/tips/defaultIsBad.html
https://stackoverflow.com/a/33307487

Reply:
@agar23 thank you for share these info with us... I believe that it's something we can discuss for the upcoming verdaccio version.. I  will really read more about it, but please could you leave it as default for now? Then if we decide to do as you suggested,  we can add this change in all components 😉

Reply:
sounds good 😄